app/views/admin_mailer/feedback.text.erb

```diff
@@ -1,9 +1,17 @@
 <% content_for :message do %>
-Here's some user feedback on the archive!  
-<% unless ArchiveConfig.REVISION.blank? %>
-
-Archive revision: (rev. <%= ArchiveConfig.REVISION %>)
-<% end %>
+Here's some user feedback on the archive!
```

Comment:
Capital A in archive to match the HTML email? Also, same comment as above regarding shortening the logic for username.